packages/gestalt/src/Masonry.js

```diff
@@ -181,9 +181,7 @@ export default class Masonry<T: { +[string]: mixed }> extends ReactComponent<Pro
 
   positionStore: Cache<T, Position>;
 
-  insertAnimationFrame: AnimationFrameID;
-
-  measureTimeout: TimeoutID;
```

Comment:
@jackhsu978 what happened to this `measureTimeout`

Reply:
Was it dead code?

Reply:
yes, it's dead code